tests/cypress/page-object/datePicker.ts

```diff
@@ -1,5 +1,5 @@
 export class DatePicker {
-    private getTodayDate(separator: string): string {
+    protected getTodayDate(separator: string): string {
```

Comment:
I know that it's already merged but it could be nice to have qant:pickers_datepicker as a parameter instead of an harcoded value in getDatePicker()